src/types/bookmarks.ts

```diff
@@ -0,0 +1,23 @@
+export enum BookmarkTypes {
+  folder = 'folder',
+  shabad = 'file',
+  bani = 'book-open'
+}
+export enum BookmarksNavigatorRoutes {
+  bookmarks = 'BookmarksList'
+}
+export type BookmarksNavigatorParams = {
+  [BookmarksNavigatorRoutes.bookmarks]: {folder?: string, folderData: FolderData[]},
+}
+
+export type FolderData = {
+  folderName: string,
+  type: BookmarkTypes,
```

Comment:
If it is a folder, why would it have a type?

Reply:
I've forgotten what I was thinking for this

Reply:
oh ok i remember now, i think this was for the icon maybe?